src/entity/BaseEntity.ts

```diff
@@ -0,0 +1,6 @@
+import { BaseEntity as TypeOrmBaseEntity, PrimaryGeneratedColumn } from 'typeorm';
+
+export abstract class BaseEntity extends TypeOrmBaseEntity {
+  @PrimaryGeneratedColumn()
```

Comment:
Consider adding ﻿@CreateDateColumn(), ﻿@UpdateDateColumn(), and ﻿@DeleteDateColumn() decorators to new columns. This will provide time tracking features for all entities extending ﻿BaseEntity, enabling better auditing and data management